src/task2.java

```diff
@@ -0,0 +1,10 @@
+public class task2 {
+    public static void main(String...args) {
+        int sideA = 10;
+        int sideB = 22;
+        int perimeter = (sideA + sideB) * 2;
+        int square = sideA * sideB;
+            System.out.println("perimeter=" + perimeter + ";" + "Square=" + square);
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"